src/modal.js

```diff
@@ -18,6 +18,7 @@ export class Modal {
     });
   }
   showingChanged(newValue){
+    if (!this.modal) return;
```

Comment:
Have you confirmed you can hit this line with `this.modal` being being null or undefined?  Just want to make sure.  Also if you want to add a test to ensure that `this.showing` that is set is passed in as an option or some other missing test coverage that might be nice :)